pkg/ktime/ktime_windows_test.go

```diff
@@ -25,3 +25,8 @@ func TestBoottime(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Greater(t, ktime1.UnixMilli(), int64(0))
 }
+
+func TestKernelime(t *testing.T) {
```

Comment:
looks like a small typo
```suggestion
func TestKernelTime(t *testing.T) {
```